RecoPixelVertexing/PixelTriplets/plugins/BrokenLineFitOnGPU.h

```diff
@@ -84,9 +98,21 @@ __global__ void kernel_BLFastFit(Tuples const *__restrict__ foundNtuplets,
     auto dz = hhp->zGlobal(hitId[hitsInFit - 1]) - hhp->zGlobal(hitId[0]);
     float ux, uy, uz;
 #endif
-    for (unsigned int i = 0; i < hitsInFit; ++i) {
-      auto hit = hitId[i];
+
+    float incr = std::max(1.f, float(nHits) / float(hitsInFit));
+    float n = 0;
+    // int jold = -1;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
was in an assert (that has been probably removed).